### PR TITLE
new icds env config

### PR DESCRIPTION
**Pull request by snopoke**
@dimagi/scale-team FYI

### CODE REVIEW

File: environments/icds-cas/app-processes.yml [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+      num_processes: 1
+    UpdateUserSyncHistoryPillow:
+      num_processes: 1
+    UserCacheInvalidatePillow:
+      num_processes: 1
+  'pillow10':
+  'pillow11':
+  'pillow12':
+  'pillow13':
+  'pillow14':
```
Comment (emord) [inline environments/icds-cas/app-processes.yml]: We'll probably need to add some partitions to kafka for both cases and forms. Should add this to the migration doc?

File: environments/icds-cas/postgresql.yml [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+      hq_acceptable_standby_delay: 1800
+    - django_alias: icds-ucr-standby2
+      name: commcarehq_icds_aggregatedata
+      host: pgucrstandby1
+      create: False
+      django_migrate: False
+      hq_acceptable_standby_delay: 1800
+    - django_alias: icds-ucr-standby3
+      name: commcarehq_icds_aggregatedata
+      host: pgucrstandby
```
[the remark this answers is not in the document]
Comment (snopoke) [reply environments/icds-cas/postgresql.yml]: 5d1a871

File: environments/icds-cas/public.yml [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+    'city': "Cambridge"
+    'region': "MA"
+    'postal_code': "02139"
+    'country': "US"
+    'phone_number': "(617) 649-2214"
+    'email': "accounts@dimagi.com"
+    'website': "http://www.dimagi.com"
+  INVOICE_PREFIX: INC-
+  INVOICE_STARTING_NUMBER: 5000
+  KAFKA_URL: "{{ groups.kafka0.0 }}:9092"
```
[the remark this answers is not in the document]
Comment (snopoke) [reply environments/icds-cas/public.yml]: that hasn't been merged yet: https://github.com/dimagi/commcare-cloud/pull/2020

**Comment by snopoke** [comment]
Closing in favor of https://github.com/dimagi/commcare-cloud/pull/2050

**Pull request closed without merging**